### DISCUSSION TITLE
请问一下,使用snapshot快照后如何禁止自动回滚

### DISCUSSION THREAD

**Discussion by allworldg** [nshen/learn-neovim-lua#64] [Q&A] [answered]
我发现像@nshen 大佬这样设置好快照snapshot后,每次启动nvim都会尝试回滚。是否有方法能禁止自动回滚？

**Comment by nshen** [comment, marked as answer]
https://github.com/nshen/learn-neovim-lua/blob/7af70083eaf469fccb2ad601eaeabac150d080e5/lua/plugins.lua#L153-L154

这行删掉就不会回滚了，v2分支也没有回滚

相关讨论
https://github.com/nshen/learn-neovim-lua/discussions/51

**Reply by allworldg** [reply]
嗯感谢大佬回复，这个我知道。我以为能在不删除的前提下设置不自动触发回滚，结果看了下issue开发者就是设计成自动回滚。 只能删除了。
